state_solver/tire.py

```diff
@@ -53,20 +53,57 @@ def lateral_pacejka(self, inclination_angle:float, normal_force:float, slip_angl
         # NOTE: 2/3 multiplier comes from TTC forum suggestions, including from Bill Cobb
         test_condition_multiplier = 2/3
         return test_condition_multiplier * (D * math.sin(C * math.atan(Bx1 - E * (Bx1 - math.atan(Bx1)))) + V) * multiplier
+    
+    warnings.filterwarnings("error")
+    def longitudinal_pacejka(self, normal_force:float, SR:float):
+        SR = SR * 100
+        FZ = normal_force / 1000
+        [b0, b1, b2, b3, b4, b5, b6, b7, b8, b9, b10, b11, b12, b13] = self.longitudinal_coeffs
+        C = b0
+        D = FZ * (b1 * FZ + b2)
+        try:
+            BCD = (b3 * FZ**2 + b4 * FZ) * np.exp(-1 * b5 * FZ)
+        except:
+            return 0.0001
```

Comment:
eek we should look into better methods of 0 handling

Reply:
@rhorvath02 updated this now I believe